src/pages/RFOX/components/Overview/StakingInfo.tsx

```diff
@@ -115,11 +117,21 @@ export const StakingInfo: React.FC<StakingInfoProps> = ({
       timeInPoolSeconds === 0n ? 'N/A' : formatSecondsToDuration(Number(timeInPoolSeconds)),
   })
 
+  const arbitrumAsset = useAppSelector(state => selectAssetById(state, arbitrumAssetId))
+
   const handleGetAssetClick = useCallback(
     (assetId: AssetId) => () => {
+      const buyAsset = selectAssetById(store.getState(), assetId)
+      if (!arbitrumAsset || !buyAsset) return
+
       navigate(`/trade/${assetId}/${arbitrumAssetId}/0`)
+
+      dispatch(tradeInput.actions.setSellAsset(arbitrumAsset))
+      dispatch(tradeInput.actions.setBuyAsset(buyAsset))
+      dispatch(tradeInput.actions.setSellAccountId(stakingAssetAccountId))
```

Comment:
was this part of the spec? feels like account 0 would have a much higher chance of actually being something the user could trade from.

Reply:
Happy with whichever feels the most sane from a user-perspective tbh, not particularly attached or opinionated on this one! 
Just felt more natural to sync, but counter-argument sounds sane too, agree more chances of account 0 having funds.
Although counter argument to the counter-argument is if you selected that account in particular, chances are it's well-funded?
Urghh, heuristics.

Reply:
Yeah, just a thought and I didn't put much more thought into it than seems like account 0 haz mo monies usually lol. No super strong feelings